packages/flutter/lib/src/widgets/icon.dart

```diff
@@ -231,14 +233,30 @@ class Icon extends StatelessWidget {
   /// specified, either directly using this property or using [Directionality].
   final TextDirection? textDirection;
 
+  /// Whether the size is affected by the context's text scaler.
```

Comment:
maybe: Whether to scale the size of this widget using the ambient `MediaQuery`'s `TextScaler`. ?

Reply:
Done